src/widgetastic_patternfly4/select.py

```diff
@@ -24,88 +22,25 @@ class Select(GenericLocatorWidget):
     ITEM_LOCATOR = (
         ".//button[contains(@class, 'pf-c-select__menu-item')" " and normalize-space(.)={}]"
     )
-
-    @contextmanager
-    def opened(self):
-        """A context manager to open and then close a Select."""
-        self.open()
-        yield
-        self.close()
-
-    @property
-    def is_open(self):
-        """Returns a boolean detailing if Select is open."""
-        return "pf-m-expanded" in self.browser.classes(self)
-
-    def open(self):
-        """Opens a Select."""
-        if not self.is_open:
-            self.browser.click(self.BUTTON_LOCATOR)
-
-    def close(self):
-        """Closes a Select."""
-        if self.is_open:
-            self.browser.click(self)
-
-    @property
-    def items(self):
-        """Returns a list of all Select items as strings."""
-        with self.opened():
-            result = [self.browser.text(el) for el in self.browser.elements(self.ITEMS_LOCATOR)]
-        return result
-
-    @property
-    def enabled_items(self):
-        """Returns a list of all enabled Select items as strings."""
-        result = []
-        with self.opened():
-            for el in self.browser.elements(self.ITEMS_LOCATOR):
-                item_value = self.browser.text(el)
-                if self.item_enabled(item_value):
-                    result.append(item_value)
-        return result
-
-    def has_item(self, item):
-        """Returns whether the items exists.
-
-        Args:
-            item: item name
-
-        Returns:
-            Boolean - True if enabled, False if not.
-        """
-        return item in self.items
+    SELECTED_ITEM_LOCATOR = (
+        ".//span[contains(@class, 'ins-c-conditional-filter')" " and normalize-space(.)={}]"
```

Comment:
Why do we have the double quotes here in the middle of this line? Can't this just be:

`".//span[contains(@class, 'ins-c-conditional-filter') and normalize-space(.)={}]"`